src/cswinrt/code_writers.h

```diff
@@ -6483,11 +6483,11 @@ bind<write_type_name>(type, typedef_name_type::CCW, true)
                 {
                     handler = (%) =>
```

Comment:
I assume we are intentionally not doing anything similar to _state.eventInvoke below as I assume it is done below for performance reasons only?

Reply:
not sure what you're asking - this is Ujjwal's event source optimization logic

Reply:
In the reflection version of this EventInvoke that this overrides, there was changes to set _state.EventInvoke and store it as part of the state.  I was wondering whether it is intentional that we aren't doing the same with handler here which is the same as eventInvoke there.  Is it because this is just a function lamda in code while that was an expression that had to be computed again?